tools/run_code_checks.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -x
```

Comment:
This script is not covered by codeChecks.py? 
This is specific to this project... 
If this is not a temporary script, why this functionality included in the rest of the python utility?